modules/oe_theme_helper/config/install/oe_theme_helper.internal_domains.yml

```diff
@@ -0,0 +1 @@
+internal_domain: ''
```

Comment:
EWCMS should set the value to a regex that matches the constraints set by COMM. SOmething like: 
http(s)?:\/\/[a-z]*\.?europa.eu(\/.*)*